src/ResolvedPeers.cc

```diff
@@ -13,32 +13,29 @@
 #include "ResolvedPeers.h"
 #include "SquidConfig.h"
 
+/* ResolvedPeers */
+
```

Comment:
Pointless addition. The namespace of every method documents the scope. Please remove.

```suggestion
```

Reply:
This cc module also has PeerConnectionPointer methods (with a corresponding separating comment). I think that If we follow this style, we should keep both comments or none.

Reply:
Yes, it should not have either. This type of comment is an artifact of coding practices to document C and older language module boundaries. Those boundaries are self-documenting in modern languages like C++.

Reply:
These visual "section" markings help navigate the files that contain multiple class definitions. The "inlined" difference in class names is not enough for that purpose.

Long time ago, Amos has banned repeated empty lines that served a similar section-delimiting purpose. Please leave us at least these marks! I do not insist on others adding them, but they help me and, I speculate, some other developers.

Reply:
What is it delimiting though? ... that code following "/* Foo */"  are in a Foo:: namespace. One would have to be blind not to notice from the method names themselves, or the filename at least.

A comment like this has some utility delimiting sections of code which is *misplaced* and should be in another cc unit. Which is not the case for these comments AFAIK.

Reply:
Roughly speaking, these "section start" comments (not uncommon in Squid code) announce the start of a class-specific section in a file that has multiple classes. When I scroll through such a file, they help me notice that I crossed into another section (because they are visually different from any method start and most method descriptions). They have other, secondary benefits as well, such as highlighting classes that have only one or two small methods, which, sometimes, helps readers understand the class role in the code.

The first "section start" comment does not really delimit anything -- it announces the first section. The first section comment is primarily useful for consistency sake -- all sections in this file should have these comments -- but it is also marginally useful for warning the reader that this file has multiple sections.

This formatting trick has its cons, of course. For example, the section boundaries/names are difficult to define precisely for non-class code. Also, the section start comments duplicate information.

As most formatting conventions, it is impossible to formally prove that these comments should stay. If you insist on eliminating them for some reason, or if you are going to police every section start comment (or lack of thereof), then I am not going to fight for their preservation.

Reply:
Like I said its an antiquated coding style. Properly scoped units would only have one of these at the top saying the same thing as the filename - which makes it pointless. Exactly what we are seeing with this comment.

Reply:
Done (91a2607).